modules/ivc_champva/app/services/ivc_champva/file_uploader.rb

```diff
@@ -24,6 +28,22 @@ def handle_uploads
 
     private
 
+    def insert_form(pdf_file_path, response_status)
+      pega_status = response_status.first == 200 ? 'Submitted' : nil
+      IvcChampvaForm.create!(
+        form_uuid: @metadata['uuid'],
+        email: validate_email(@metadata&.dig('primaryContactInfo', 'email')),
+        first_name: @metadata&.dig('primaryContactInfo', 'name', 'first'),
+        last_name: @metadata&.dig('primaryContactInfo', 'name', 'last'),
```

Comment:
should `primaryContactInfo` be camelcase here? on line 68 in this file we're using snake case